CRM/Utils/Address/BatchUpdate.php

```diff
@@ -266,7 +283,26 @@ public function processContacts($processGeocode, $parseStreetAddress) {
         }
       }
     }
-
+    foreach ($unparsedContactIds as $eachContactId) {
```

Comment:
This isn't needed here. We already have the address ID from the DAO above. The simple thing to do is to set the manual geocode to TRUE where you are declaring the unparsedContactIds on L253